src/databases/OpenPMD/OpenPMDClasses/PMDField.C

```diff
@@ -768,6 +779,7 @@ PMDField::SetFieldBoundary(char * name,
             }
             //cerr << this->fieldBoundary[iLabel] << endl;
         }
+		delete buffer;
```

Comment:
Replace tabs with spaces and indent properly. You can also add a blank line before the delete.